cmake/FindMDL_SDK.cmake

```diff
@@ -0,0 +1,30 @@
+#Looks for environment variable:
```

Comment:
We need the standard copyright header here -- you can just grab that from another CMake file